packages/ckeditor5-utils/src/toarray.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/toarray
+ */
+
+/**
+ * Transforms any value to an array, if it is not already one. If provided value is already an array, it is returned unchanged.
+ *
+ * @param {*} [data] Value to transform to an array.
+ * @returns {Array} Array created from data.
+ */
+export default function toArray( data = [] ) {
```

Comment:
Quick question, as everything else looks very well: Does the default value is needed?

From the changes, I didn't see that we cared for this particular case earlier and we might not need to.

I'm asking here because it does _slightly_ change the logic of this utility:

```javascript
// A. no default:
( toArray() ).length === 1;

// B. with default:
( toArray() ).length === 0;
```

The current code will work with as in the (A) case since we always pass a variable. Passing an `undefined` value would be an error most probably and probably should fail later on. Having a default value would hide this error because other logic would get an empty array.

Reply:
You are right: the default `[]` is not needed. This new `toArray()` helper should behave exactly the same as the code before the changes 👍🏻